flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/HDFSDataStream.java

```diff
@@ -91,14 +88,13 @@ protected void doOpen(Configuration conf,
       if(hdfs instanceof LocalFileSystem) {
         hdfs = ((LocalFileSystem)hdfs).getRaw();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "hdfs". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1226)